transient/transient.py

```diff
@@ -326,20 +327,33 @@ def run(self) -> None:
             logging.error("QEMU Process has died. Exiting")
             return self.__post_run(qemu_returncode)
 
+        sshfs_threads = []
         for shared_spec in self.config.shared_folder:
             assert self.ssh_config is not None
             local, remote = shared_spec.split(":")
 
             # The user almost certainly doesn't intend to pass a relative path,
             # so make it absolute
             absolute_local_path = os.path.abspath(local)
-            sshfs.do_sshfs_mount(
-                connect_timeout=self.config.ssh_timeout,
-                ssh_config=self.ssh_config,
-                local_dir=absolute_local_path,
-                remote_dir=remote,
-                local_user=self.__current_user(),
+            sshfs_kwargs = {
+                "connect_timeout": self.config.ssh_timeout,
+                "ssh_config": self.ssh_config,
+                "local_dir": absolute_local_path,
+                "remote_dir": remote,
+                "local_user": self.__current_user(),
+            }
+
+            # Slamming the server with 20 connections at once is a good way to break things:
+            if len(sshfs_threads) > 7:
```

Comment:
I'd like the pulled out to a global `MAX_CONCURRENT_SSHFS`

Reply:
Fixed in my most recent commit.